src/app/app.component.ts

```diff
@@ -9,10 +11,31 @@ import { LanguageSwitchService } from './services/language-switch/language-switc
     styleUrls: ['./app.component.scss'],
 })
 export class AppComponent implements OnInit {
+    showHead = false;
+
     constructor(
         readonly translate: TranslateService,
-        readonly languageSwitchService: LanguageSwitchService,
-    ) {}
+        private readonly languageSwitchService: LanguageSwitchService,
+        private readonly router: Router,
+    ) {
+        this.router.events.forEach(event => {
```

Comment:
replase event with {url} and use just URL below

Reply:
rework this function like this.showHeader = this.router.events.some(({url})=> );